build.gradle

```diff
@@ -25,6 +25,7 @@ allprojects {
 	group "org.springframework.boot"
 
 	repositories {
+		mavenLocal() // for Tracing snapshots
```

Comment:
We can't merge this as it'll fail on CI. The plan is to switch to snapshots for Tracing and Micrometer tomorrow so this will shortly become unnecessary anyway.

Reply:
Removed